packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -153,48 +204,11 @@ const ModalContent = forwardRef((props, ref) => {
           the aria-label value when the modal is opened
         */
       }
-      <div
-        {...customProps}
-        tabIndex={platformIsiOS || isCalledFromNotificationDialog ? '-1' : '0'}
-        aria-label={ariaLabel}
-        aria-labelledby={ariaLabelledBy}
-        aria-describedby={ariaDescribedBy}
-        className={modalClassName}
-        role={role}
-        ref={ref}
-      >
-        <div className={modalContainerClassName} ref={setModalFocusElementRef} data-terra-abstract-modal-begin tabIndex="-1">
-          {(!isCalledFromNotificationDialog) && (
-            <FormattedMessage id="Terra.AbstractModal.BeginModalDialog">
-              {text => {
-                // In the latest version of react-intl this param is an array, when previous versions it was a string.
-                let useText = text;
-                if (Array.isArray(text)) {
-                  useText = text.join('');
-                }
-                return (
-                  <VisuallyHiddenText text={useText} />
-                );
-              }}
-            </FormattedMessage>
-          )}
-          {children}
-          {(!isCalledFromNotificationDialog) && (
-            <FormattedMessage id="Terra.AbstractModal.EndModalDialog">
-              {text => {
-                // In the latest version of react-intl this param is an array, when previous versions it was a string.
-                let useText = text;
-                if (Array.isArray(text)) {
-                  useText = text.join('');
-                }
-                return (
-                  <VisuallyHiddenText text={useText} />
-                );
-              }}
-            </FormattedMessage>
-          )}
-        </div>
-      </div>
+      {
+        shouldTrapFocus
+          ? <FocusOn onClickOutside={closeOnOutsideClick ? onRequestClose : null}>{modalContent}</FocusOn>
+          : <>{modalContent}</>
```

Comment:
You should not need a React fragment around a div element.

Reply:
If I remove this fragment it is throwing the below error.

<img width="1208" alt="Screenshot 2024-02-08 at 9 39 23 PM" src="https://github.com/cerner/terra-framework/assets/126655256/50ee5096-8f64-4ba5-a71b-32aa12e85e1d">